ios/RNEosEcc.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.description  = <<-DESC
                   RNEosEcc
                    DESC
-  s.homepage     = ""
+  s.homepage     = "https://fioprotocol.io/"
```

Comment:
Change to: "https://github.com/EvaCoop/react-native-eos"

This copies what is at: https://github.com/EvaCoop/react-native-eos/blob/master/RNEosEcc.podspec

Reply:
@ericbutz the develop branch already has the value (`https://github.com/author/RNEosEcc.git`) - https://github.com/fioprotocol/fiosdk_typescript/blob/develop/ios/RNEosEcc.podspec#L9

Reply:
new PR https://github.com/fioprotocol/fiosdk_typescript/pull/120